gcp/appengine/blog/content/posts/using-the-determineversion-api.md

```diff
@@ -0,0 +1,184 @@
+                "published": "2021-05-20T00:00:30.166614Z",
+                "references": [
+                    {
+                    "type": "REPORT",
+                    "url": "https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=34461"
+                    }
+                ],
+                "affected": [
+                    {
+                    "package": {
+                        "name": "libxml2",
+                        "ecosystem": "OSS-Fuzz",
+                        "purl": "pkg:generic/libxml2"
+                    },
+                    "ranges": [
+                        {
+                        "type": "GIT",
+                        "repo": "https://gitlab.gnome.org/GNOME/libxml2.git",
+                        "events": [
+                            {
+                            "introduced": "6c128fd58a0e4641c23a345d413672494622db1b"
+                            }
+                        ]
+                        }
+                    ],
+                    "versions": [
+                        "CVE-2021-3541",
+                        "v2.9.11",
+                        "v2.9.12",
+                        "v2.9.13",
+                        "v2.9.14",
+                        "v2.10.0",
+                        "v2.10.1",
+                        "v2.10.2",
+                        "v2.10.3",
+                        "v2.10.4",
+                        "v2.11.0",
+                        "v2.11.1",
+                        "v2.11.2",
+                        "v2.11.3",
+                        "v2.11.4"
+                    ],
+                    "ecosystem_specific": {
+                        "severity": "HIGH"
+                    },
+                    "database_specific": {
+                        "source": "https://github.com/google/oss-fuzz-vulns/blob/main/vulns/libxml2/OSV-2021-777.yaml"
+                    }
+                    }
+                ],
+                "schema_version": "1.4.0"
+                }
+            ]
+            }
+            ```
+5. Finally, we consider the response and draw conclusions. 
+
+Our copy of libxml2 returned a number of versions with similar confidence scores. 
+- Version 2.11.3, 0.718
+- Version 2.11.4, 0.718
+- Version 2.11.2, 0.707
+- Version 2.11.0, 0.697
+
+To be sure we have caught any potential vulnerabilities, we could make further queries for these other versions and see if any other vulnerabilities are found. 
+
+Additionally, we can be fairly confident that the vulnerability that we found in our sample API call ([OSV-2021-777](https://osv.dev/vulnerability/OSV-2021-777)) is in our local copy of libxml2. This is because there is overlap between the other likely versions of libxml2 and the versions vulnerable to OSV-2021-777.
+
+By running one Go module and making one additional API call, we now are fairly confident that my local version of libxml2 has known vulnerability OSV-2021-777.
```

Comment:
Is this go module referring to the `indexer-api-caller`?, the term "go module" sounds a bit weird here.

I would just have this say "by making two API calls, first to determineversion, and then to query" or something similar.

If you would like to keep the mention of the tool, then something like: "by running out helper tool and making one additional API call"

Reply:
Yes, that was what I was referring to.